.github/workflows/build.yml

```
@@ -0,0 +1,44 @@
+name: Build AAR
+
+on:
+  workflow_dispatch:
+    branches:
```

actionlint (1.7.12)

[error] expected "inputs" key for "workflow_dispatch" section but got "branches"

(syntax-check)